BluePic-Server/Sources/Server/Routes.swift

```diff
@@ -114,18 +108,19 @@ func defineRoutes() {
       } else {
         Log.error("Status error code or nil reponse received from MCA server.")
         if let resp = resp {
-          Log.error("Status code: \(resp.statusCode)")
-          if let rawUserData = try? BodyParser.readBodyData(with: resp) {
-            #if os(OSX)
-              let str = NSString(data: rawUserData as Data, encoding: String.Encoding.utf8.rawValue)
-            #else
-              let str = NSString(data: rawUserData, encoding: NSUTF8StringEncoding)
-            #endif
-            print("Response from MCA server: \(str)")
-          }
+            Log.error("Status code: \(resp.statusCode)")
+            do {
```

Comment:
Yeah, this bit is not as clear cut as an action item. But I I see this throughout the code. It will improve code-readability and in turn perhaps prevent bugs. Hard to add as an actionable 'issue'.